<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicDownloadWidget</class>
 <widget class="QDialog" name="MusicDownloadWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>460</width>
     <height>220</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="topTitleName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:&quot;#FFFFFF&quot;;font-weight:bold</string>
   </property>
   <property name="text">
    <string>下载窗口</string>
   </property>
  </widget>
  <widget class="QToolButton" name="topTitleCloseButton">
   <property name="geometry">
    <rect>
     <x>458</x>
     <y>22</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QWidget" name="backgroundMask" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>45</y>
     <width>460</width>
     <height>195</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{color:#666666;background:rgba(255,255,255,200);}
QLabel{background:transparent;}</string>
   </property>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>存储路径：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="downloadPathEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{font-size:12px;color:#BFBFBF; font-weight:normal;border: 1px solid gray;}QLineEdit::hover{ border: 1px solid #AAAAFF;}</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pathChangedButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>120</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-radius:1px;background-color:#80B7F1;color:white;font-size:12px;}</string>
    </property>
    <property name="text">
     <string>更改目录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="settingButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>120</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-radius:1px;background-color:#80B7F1;color:white;font-size:12px;}</string>
    </property>
    <property name="text">
     <string>下载设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="downloadButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>147</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-radius:5px;background-color:#80B7F1;color:white;font-size:17px;font-weight:bold;}</string>
    </property>
    <property name="text">
     <string>下载</string>
    </property>
   </widget>
   <widget class="QLabel" name="downloadName">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>2</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:22px;font-weight:bold;</string>
    </property>
    <property name="text">
     <string>XXXXXXXXXXXXXXXXXXX</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>401</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="music">
     <widget class="QLabel" name="musicLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>选择音质：</string>
      </property>
     </widget>
     <widget class="QLabel" name="iconHD">
      <property name="geometry">
       <rect>
        <x>145</x>
        <y>37</y>
        <width>30</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="iconSD">
      <property name="geometry">
       <rect>
        <x>145</x>
        <y>62</y>
        <width>30</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonSD">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>70</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">            QRadioButton{ background-color:transparent; }
            QRadioButton::indicator::unchecked{ image: url(:/control/radiobutton_unchecked); }
            QRadioButton::checked { color: rgb(230,115,0); } 
            QRadioButton::indicator:checked { image: url(:/control/radiobutton_checked);}</string>
      </property>
      <property name="text">
       <string>高清品质</string>
      </property>
     </widget>
     <widget class="QLabel" name="informationHD">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>35</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>XXX/XXXXXX/XXXXX</string>
      </property>
     </widget>
     <widget class="QLabel" name="informationST">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>XXX/XXXXXX/XXXXX</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonST">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>70</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">            QRadioButton{ background-color:transparent; }
            QRadioButton::indicator::unchecked{ image: url(:/control/radiobutton_unchecked); }
            QRadioButton::checked { color: rgb(230,115,0); } 
            QRadioButton::indicator:checked { image: url(:/control/radiobutton_checked);}</string>
      </property>
      <property name="text">
       <string>压缩音质</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonHD">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>35</y>
        <width>70</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">            QRadioButton{ background-color:transparent; }
            QRadioButton::indicator::unchecked{ image: url(:/control/radiobutton_unchecked); }
            QRadioButton::checked { color: rgb(230,115,0); } 
            QRadioButton::indicator:checked { image: url(:/control/radiobutton_checked);}</string>
      </property>
      <property name="text">
       <string>标准品质</string>
      </property>
     </widget>
     <widget class="QLabel" name="informationSD">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>60</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>XXX/XXXXXX/XXXXX</string>
      </property>
     </widget>
     <zorder>musicLabel</zorder>
     <zorder>radioButtonSD</zorder>
     <zorder>informationHD</zorder>
     <zorder>informationST</zorder>
     <zorder>radioButtonST</zorder>
     <zorder>radioButtonHD</zorder>
     <zorder>informationSD</zorder>
     <zorder>iconSD</zorder>
     <zorder>iconHD</zorder>
    </widget>
    <widget class="QWidget" name="movie">
     <widget class="QLabel" name="movieLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>视频资源：</string>
      </property>
     </widget>
     <widget class="QLabel" name="informationSD_MV">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>35</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>XXX/XXXXXX/XXXXX</string>
      </property>
     </widget>
     <widget class="QLabel" name="iconHD_MV">
      <property name="geometry">
       <rect>
        <x>145</x>
        <y>11</y>
        <width>30</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="iconSD_MV">
      <property name="geometry">
       <rect>
        <x>145</x>
        <y>36</y>
        <width>30</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="informationHD_MV">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>XXX/XXXXXX/XXXXX</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonHD_MV">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>70</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">            QRadioButton{ background-color:transparent; }
            QRadioButton::indicator::unchecked{ image: url(:/control/radiobutton_unchecked); }
            QRadioButton::checked { color: rgb(230,115,0); } 
            QRadioButton::indicator:checked { image: url(:/control/radiobutton_checked);}</string>
      </property>
      <property name="text">
       <string>标准品质</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonSD_MV">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>35</y>
        <width>70</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">            QRadioButton{ background-color:transparent; }
            QRadioButton::indicator::unchecked{ image: url(:/control/radiobutton_unchecked); }
            QRadioButton::checked { color: rgb(230,115,0); } 
            QRadioButton::indicator:checked { image: url(:/control/radiobutton_checked);}</string>
      </property>
      <property name="text">
       <string>高清品质</string>
      </property>
     </widget>
     <zorder>movieLabel</zorder>
     <zorder>informationSD_MV</zorder>
     <zorder>informationHD_MV</zorder>
     <zorder>radioButtonHD_MV</zorder>
     <zorder>radioButtonSD_MV</zorder>
     <zorder>iconSD_MV</zorder>
     <zorder>iconHD_MV</zorder>
    </widget>
   </widget>
   <zorder>label2</zorder>
   <zorder>downloadName</zorder>
   <zorder>downloadPathEdit</zorder>
   <zorder>pathChangedButton</zorder>
   <zorder>settingButton</zorder>
   <zorder>downloadButton</zorder>
   <zorder>stackedWidget</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
